nixos/modules/virtualisation/google-compute-config.nix

```diff
@@ -1,5 +1,5 @@
 { ... }:
 
 {
-  imports = [ <nixpkgs/nixos/modules/virtualisation/google-compute-image.nix> ];
+  imports = [ ./google-compute-image.nix ];
```

Comment:
I think this simply acts as a template copied into the image to `/etc/nixos/configuration.nix`, so using `<nixpkgs>` here should be fine and required, shouldn't it?

Reply:
Okay, maybe we should rename this file to `google-compute-config.nix.tpl` in that case to make it clear that it's not meant to be used?

Reply:
Maybe swapping `tpl` and `nix`, so `google-compute-config.nix` (as it's still valid nix code) - but this should then be consistent with everything else using `make-disk-image.nix` - and I wouldn't bother for that with 18.09.